### DISCUSSION TITLE
⌛ 🔗 Easy Digital Download block support

### DISCUSSION THREAD

**Discussion by Nantris** [elementor/elementor#23968] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

In Easy Digital Downloads 3 there are Gutenberg blocks required for certain features of the site. Unfortunately Elementor provides no way to use these, nor does it provide EDD-specific blocks is the case for WooCommerce.

Unfortunately this means that the only other options are to use the limited shortcodes which lack features, or to pay for yet another plugin (PostX) to accommodate this.

### Describe the solution you'd like

Provide a way to use Gutenberg blocks or provide EDD-specific login and confirmation-page blocks.

EDD should receive the same attention that WooCommerce does rather than the current situation where these two paid plugins then necessitate a third paid plugin.

### Describe alternatives you've considered

There is no alternative besides paying for a third plugin, which is why I hope the Elementor team will take this suggestion to heart.

### Additional context

![image](https://github.com/elementor/elementor/assets/6835891/1d477fd3-1deb-473c-b6c8-32fafbfe2c44)

**Comment by gerardreches** [comment]
+1 to this. I want to remove this warning, but I need Elementor to include a widget for it because I'm using Elementor, not Gutenberg.

![image](https://github.com/elementor/elementor/assets/7549815/ef07dcc4-3460-41d9-867b-c96509a083e0)

**Comment by Nantris** [comment]
I really hope this will be addressed?

**Comment by cklosowski** [comment]
> [!NOTE]  
> To the Elementor team: If you decide to move forward on this and need any help from our end, please feel free to reach out to me. Our team is happy to assist you in any way we can.
